src/style.css

```diff
@@ -0,0 +1,49 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
```

Comment:
Try to avoid `*` selector. In these tasks it doesn’t influence performance, but in bigger projects, it can load browser